### PR TITLE
Updating config.toml template for tracing

### PR DESCRIPTION
**Pull request by pubudu538**
### Issues

Fixes https://github.com/wso2/product-microgateway/issues/2426
### Automation tests
 - Unit tests added: Yes/No
 - Integration tests added: Yes/No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

File: resources/conf/config.toml.template [hunk 1]
```diff
@@ -479,18 +479,18 @@ enabled = true
 [tracing]
   # Enable/Disable tracing in Choreo Connect
   enabled = false
-  # Type of tracer exporter (e.g: azure, jaeger, zipkin)
-  type = "jaeger"
+  # Type of tracer exporter (e.g: azure, zipkin). Use zipkin type for Jaeger as well.
+  type = "zipkin"
```
Comment (AmaliMatharaarachchi) [inline resources/conf/config.toml.template:483]: starting from line 500 has zipkin related configuration example already

**Comment by praminda** [comment]
@pubudu538 can you remove following from the config.toml with this PR
 
 ​https://github.com/wso2/product-microgateway/blob/main/resources/conf/config.toml#L77-L78

File: resources/conf/config.toml.template (hunk 1 above)
Comment (pubudu538) [reply resources/conf/config.toml.template:483]: We need to provide zipkin as the type for testing jaeger as well.

**Comment by pubudu538** [comment]
@praminda Removed the requested lines.

**Review by praminda — APPROVED** [review]

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment : 20211202.25

**Pull request merged**